packages/node_modules/@webex/webex-core/test/integration/spec/credentials/credentials.js

```diff
@@ -19,6 +19,25 @@ describe('webex-core', () => {
         user = u;
       }));
 
+    describe('#determineOrgId()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('generate the webex instance', () => {
+        webex = new WebexCore({
+          credentials: user.token
+        });
+
+        credentials = webex.credentials;
+      });
+
+      it('should return the OrgId of a client authenticated user', () => {
```

Comment:
Is there a way to do an integration test with the access token as well? That way we can be sure the flow of falling back to token works.

Reply:
There is no way for us to generate a real user auth token for integration testing that I'm aware of. Let me know if there's a way and I'll implement it.